lib/rubocop/cop/sorbet/sigils/valid_sigil.rb

```diff
@@ -75,6 +76,25 @@ def check_sigil_present(sigil)
           false
         end
 
+        def suggested_strictness_level(minimum_strictness, suggested_strictness)
+          # if no minimum strictness is set (eg. using Sorbet/HasSigil without config) then
+          # we always use the suggested strictness which defaults to `false`
+          return suggested_strictness unless minimum_strictness
+
+          # special case: if you're using Sorbet/IgnoreSigil without config, we should recommend `ignore`
+          return "ignore" if minimum_strictness == "ignore" && cop_config['SuggestedStrictness'].nil?
+
+          # if a minimum strictness is set (eg. you're using Sorbet/FalseSigil)
+          # we want to compare the minimum strictness and suggested strictness. this is because
+          # the suggested strictness might be higher than the minimum (eg. if you want all new files
+          # at a higher strictness level, without having to migrate existing files at lower levels).
+
+          suggested_level = STRICTNESS_LEVELS.index(suggested_strictness)
+          minimum_level = STRICTNESS_LEVELS.index(minimum_strictness)
```

Comment:
Can we do sanity check on these levels? Since `suggested_strictness` and `minimum_strictness` are user supplied values, there is a chance that they might be wrong, so there is a chance that these values end up being `-1`

Reply:
https://github.com/Shopify/rubocop-sorbet/pull/19/commits/d18620e60f7d5857dbebc348dadd25aa033f9f36